gkeys/gkeys/email_reminders.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function
+
+import os
+import re
+import smtplib
+import sys
+
+from string import Template
+
+from email.mime.text import MIMEText
+from snakeoil.demandload import demandload
+
+if sys.version_info[0] >= 3:
+    py_input = input
+    _unicode = str
+else:
+    py_input = raw_input
+    _unicode = unicode
+
+demandload(
+    "gkeys.base:Args",
+    "json:load",
+)
+
+class Emailer(object):
+    '''Send an email reminder about the status of the user's GPG key'''
+
+    def __init__(self, results, gkey):
+        uid = ''
+        if gkey.uid:
+            uid = gkey.uid[0]
+            match = re.findall(r'[\w\.-]+@[\w\.-]+', uid)
+            uid = ''
+            if match:
+                uid = match[0]
+        self.uid = uid
+        self.sub = []
+        for key in results:
+            print(results[key])
+            for specs in results[key]:
+                if specs.key == 'PUB':
+                    self.pub = specs
+                else:
+                    self.sub.append(specs)
+
+    def _send_email(self, message):
+        mail = smtplib.SMTP('smtp.gmail.com', 587)
+        mail.ehlo()
+        mail.starttls()
+        mail.login('aperivol@gmail.com', 'PASSWORD')
+        subject = "Expiring Key"
+        email_to = "justthisthing@gmail.com" #uid
+        email_from = "aperivol@gmail.com"
+        msg = MIMEText(message, 'plain')
+        msg['Subject'] = subject
+        msg['From'] = email_from
+        mail.sendmail('aperivol@gmail.com', 'justthisthing@gmail.com', msg.as_string())
+        mail.quit()
```

Comment:
Everything here is hard-coded to your initial info.  So this PR can not be merged.   Make this configurable, even add config options (to a separate email.conf file) to specify the user sendmail login info, etc. Use a config header [foo], [bar] with their key=value items so that you can specify --from foo, or --from bar and  get each of the individual settings...